charts/meep-webhook/values.yaml

```diff
@@ -15,9 +15,10 @@ deployment:
   dependency: meep-redis-master
 
 service:
-  type: NodePort
-  targetPort: 443
-  nodePort: 30006
+  type: ClusterIP
+  # type: NodePort
+  port: 443
+  # nodePort: 30006
```

Comment:
I note you kept the old ports in comment.
I think these can be safely removed now.

Reply:
Removed.